jax/api.py

```diff
@@ -26,10 +26,11 @@
 import functools
 import inspect
 import itertools as it
-import operator
+import sys
 import threading
 import weakref
```

Comment:
By the way, this is unused.  Should I remove it?

Reply:
I wouldn't do it in this PR.